### DISCUSSION TITLE
ScriptReference/EditorUserBuildSettings-selectedStandaloneTarget

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#225] [Notes]
# ScriptReference/EditorUserBuildSettings-selectedStandaloneTarget

https://docs.unity3d.com/ScriptReference/EditorUserBuildSettings-selectedStandaloneTarget.html

**Comment by mvi** [comment]
Note this is not the applied setting from switching platform, use [`activeBuildTarget`](https://docs.unity3d.com/ScriptReference/EditorUserBuildSettings-activeBuildTarget.html) instead for querying that.  
`selectedStandaloneTarget` instead is the setting currently being viewed in the EditorBuildSettings, and therefore may report a different target to the one you actually have active.
